src/chat/components/InputToolbar.tsx

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import {
+  View,
+  StyleSheet,
+  ScrollView,
+  TouchableOpacity,
+  Image,
+  StyleProp,
+  ViewStyle,
+  ImageStyle,
+} from 'react-native';
+import {
+  Composer,
+  InputToolbarProps,
+  SendProps,
+} from 'react-native-gifted-chat';
+
+interface IInputToolbar extends InputToolbarProps<any>, SendProps<any> {
+  isShowCamera?: boolean;
+  isShowPhotoGallery?: boolean;
+  togglePhotoGallery?: () => void;
+  containerStyle?: StyleProp<ViewStyle>;
+  composeWrapperStyle?: StyleProp<ViewStyle>;
+  composerTextInputStyle?: StyleProp<ViewStyle>;
+  customViewStyle?: StyleProp<ViewStyle>;
+  iconCamera?: any;
+  iconGallery?: any;
+  iconSend?: any;
+  iconStyle?: StyleProp<ImageStyle>;
+  iconSize?: number;
+  iconMargin?: number;
+}
+
+export const PressableIcon: React.FC<{
+  icon: any;
+  style?: StyleProp<ViewStyle>;
+  iconStyle?: StyleProp<ImageStyle>;
+  onPress?: () => void;
+  resizeMode?: 'contain' | 'cover' | 'stretch' | 'repeat' | 'center';
+  disabled?: boolean;
+  hitSlop?: { top?: number; right?: number; bottom?: number; left?: number };
+  size?: number;
+}> = ({
+  icon,
+  style,
+  iconStyle,
+  onPress,
+  resizeMode = 'contain',
+  disabled,
+  hitSlop,
+  size = 18,
+}) => (
+  <TouchableOpacity
+    disabled={disabled}
+    hitSlop={{ top: 10, right: 10, bottom: 10, left: 10, ...hitSlop }}
+    style={style}
+    onPress={onPress}
+  >
+    <Image
+      source={icon}
+      style={[{ width: size, height: size }, iconStyle]}
+      resizeMode={resizeMode}
+    />
+  </TouchableOpacity>
+);
+
+const InputToolbar: React.FC<IInputToolbar> = ({
+  isShowCamera = true,
+  isShowPhotoGallery = true,
+  togglePhotoGallery,
+  containerStyle,
+  composeWrapperStyle,
+  composerTextInputStyle,
+  customViewStyle,
+  iconCamera = require('../../images/camera.png'),
+  iconGallery = require('../../images/gallery.png'),
+  iconSend = require('../../images/send.png'),
+  iconStyle,
+  iconSize = 28,
+  iconMargin = 12,
```

Comment:
change to iconStyle so user can have more custom

Reply:
yes, removed iconSize, iconMargin, only need use **iconStyle**